web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -123,11 +130,49 @@ const FeedbackRequestPage = () => {
     }
   }
 
-  if (activeStep < 1 || activeStep > steps.length || !numbersOnly) {
-    return (
-      <Redirect to="/feedback/request?step=1"/>
-    );
-  }
+
+  useEffect(() => {
+    async function getMemberProfile() {
+      if (id) {
+        let res = await getMember(id, csrf);
+        let requesteeProfile =
+          res.payload.data && !res.error ? res.payload.data : undefined;
+        setRequestee(requesteeProfile ? requesteeProfile.name : "");
+      }
+    }
+    if (csrf) {
+      getMemberProfile();
+    }
+  }, [csrf, id]);
+
+  useEffect( () => {
+    async function getSuggestions() {
+      if (id === undefined || id === null) {
+        return;
+      }
+      let res = await getFeedbackSuggestion(id, csrf);
+      if (res && res.payload) {
+        console.log(res)
+        return res.payload.data && !res.error
+          ? res.payload.data
+          : undefined;
+      }
+      return null;
+    }
+    if (csrf) {
+      getSuggestions().then((res) => {
+        console.log(res);
+      });
+    }
+  },[id,csrf]);
+
+
+
+   if (activeStep < 1 || activeStep > steps.length) {
+      // return (
+      //   <Redirect to="/feedback/request?step=1"/>
+      // );
```

Comment:
We want this `<Redirect>` uncommented for now (there's another PR that will replace it eventually)